Tests/conf.json

```diff
@@ -3436,7 +3436,7 @@
         "urlscan.io": "Uses data that comes in the headers",
         "CloudConvert": "has a command that uploads a file (!cloudconvert-upload)",
         "Symantec Messaging Gateway": "Test playbook uses a random string",
-        "Cylance Protect": "Test playbook (get_file_sample_by_hash_-_cylance_protect_-_test) downloads a file",
+        "Cylance Protect": "Test playbook (get_file_sample_by_hash_-_cylance_protect_-_test) downloads a file. this problem was fixed but the test is not running any more because the integration is deprecated",
```

Comment:
in that case, remove it from the list (no reason to specify something that is not running)
```suggestion
```